test/functional/rpc_rawtransaction.py

```diff
@@ -113,21 +114,23 @@ def getrawtransaction_tests(self):
             assert_equal(self.nodes[n].getrawtransaction(txId, False), rawTxSigned['hex'])
 
             # 4. valid parameters - supply txid and 1 for verbose.
-            # We only check the "hex" field of the output so we don't need to update this test every time the output format changes.
+            # We only check the "hex" field of the output, getrawtransaction_verbosity_tests checks output
```

Comment:
> 
> But I am still unsure why the tests needs to be separated (or how it is related to previous test failure).
> 

Mostly clarity, I felt the `getrawtransaction_tests` was a bit confusing to follow, especially as I added the `verbosity=2` test cases.  Having a separate test for verbosity fields instead of smashing everything in one test block made reading more clear as well was intent.

Reply:
Fair enough.. That makes sense..